camp/camp-brooklyn/src/test/java/org/apache/brooklyn/camp/brooklyn/catalog/SpecParameterUnwrappingTest.java

```diff
@@ -61,6 +61,11 @@
 import org.apache.brooklyn.entity.stock.BasicStartable;
 
 public class SpecParameterUnwrappingTest extends AbstractYamlTest {
+    
+    // Expect app to have the following config keys already: 
+    // "application.stop.shouldDestroy", "defaultDisplayName", "quorum.running", "quorum.up", "start.latch"
+    public static final int NUM_APP_DEFAULT_CONFIG_KEYS = 5;
```

Comment:
One thing  adding config keys on the app causes is that they get set on the top-level entity when unwrapping. As you discovered, having to add this constant. Shouldn't cause problems but could be surprising in some cases. Just to keep in mind - nothing to do on this PR.